x-pack/filebeat/input/s3/input.go

```diff
@@ -314,7 +319,7 @@ func (p *s3Input) receiveMessage(svcSQS sqsiface.ClientAPI, visibilityTimeout in
 			WaitTimeSeconds:       &waitTimeSecond,
 		})
 
-	return req.Send(p.context)
+	return req.Send(p.inputCtx)
```

Comment:
When using the context here you should create a new one from it with a timeout for the call

Reply:
done! Thank you!